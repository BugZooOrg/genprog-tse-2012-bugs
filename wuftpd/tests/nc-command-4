user ftp
pass weimer@localhost
site exec /a/b
quit
