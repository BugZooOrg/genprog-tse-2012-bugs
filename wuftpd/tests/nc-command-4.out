331 Guest login ok, send your complete e-mail address as password.
230 Guest login ok, access restrictions apply.
200  (end of 'b')
221-You have transferred 0 bytes in 0 files.
221-Thank you for using the FTP service on yuki.cs.virginia.edu.
221 Goodbye.
